src/types.ts

```diff
@@ -108,3 +108,6 @@ export type ProvenanceEventTypes =
   'Set Node Size Variable' |
   'Set Select Neighbors'|
   'Set Directional Edges';
+
+export const internalFieldNames = ['_key', '_from', '_to', '_id', '_rev'] as const;
+export type InternalField = (typeof internalFieldNames)[number];
```

Comment:
Took me a few minutes to learn this idiom (pretty cool). This [SO answer](https://stackoverflow.com/a/54061487/1886928) has it without the parentheses, which seems a bit clearer to me. But this is completely new to me, so I'm just curious where you obtained the syntax for this declaration. (Just trying to learn about this new idiom; I'm not making any specific suggestion for this line of code.)

Reply:
I saw this on SO, too. I wanted to have a list we could modify to select which variables are hidden and this was the main suggestion online